python/src/BetaFactory_doc.i.in

```diff
@@ -55,31 +55,33 @@ are estimated from a sample.
 >>> factory = ot.BetaFactory()
 >>> estimated_dist = factory.build(sample)
 >>> estimated_beta = factory.buildAsBeta(sample)"
+
 // ---------------------------------------------------------------------
 
-%define OT_DistributionFactory_buildAsBeta_doc
-"Build the Beta distribution.
+%feature("docstring") OT::BetaFactory::buildAsBeta
+"Estimate the distribution as native distribution.
 
 **Available usages**:
 
+    buildAsBeta()
+
     buildAsBeta(*sample*)
 
     buildAsBeta(*param*)
 
 Parameters
 ----------
-sample : 2-d sequence of float
+sample : 2-d sequence of float,
```

Comment:
no commas at the end please, nobody does that anywhere else

Reply:
ok. I did it on all the Distribution and Factory files.